src/backends/js-testing.js

```diff
@@ -109,14 +108,16 @@ export default class JSTestingBackend extends BaseBackend {
     }
 
     async takeScreenshot (id, screenshotPath) {
-        return new Promise(async (resolve, reject) => {
-            var buffer = await requestApi(BROWSERSTACK_API_PATHS.screenshot(this.workers[id].id));
+        var buffer = await requestApi(BROWSERSTACK_API_PATHS.screenshot(this.workers[id].id));
 
-            jimp
-                .read(buffer)
-                .then(image => image.write(screenshotPath, resolve))
-                .catch(reject);
-        });
+        try {
+            var image = await jimp.read(buffer);
+
+            await image.writeAsync(screenshotPath);
+        }
+        catch (error) {
```

Comment:
I guess we can remove this `catch` section, can't we?

Reply:
We sure can!